### PR TITLE
Refactor Jobservice bench

### PR DESCRIPTION
**Pull request by jja725**
### What changes are proposed in this pull request?

Refactor job service bench to separate the file creation stage and actual distributed load. When we put file creation in prepare function, it would take a long time to prepare files and it's hard to estimate the start time for the distributed load tasks. So lots of tasks would miss start time and throw errors in cluster mode.  
And refactor TaskResultStatistics class to reuse code

### Why are the changes needed?

Please clarify why the changes are needed. For instance,

### Does this PR introduce any user facing changes?

Please list the user-facing changes introduced by your change, including

### CODE REVIEW

**Comment by jja725** [comment]
@yuzhu @ssz1997 updating the job service bench after implementing cluster mode in overmind. PTAL

File: stress/common/src/main/java/alluxio/stress/jobservice/JobServiceBenchParameters.java [hunk 1]
```diff
@@ -40,7 +40,7 @@
 
   @Parameter(names = {"--file-size"},
       description = "The size of a file for the Create op, allowed to be 0. (0, 1m, 2k, 8k, etc.)")
-  public int mFileSize = 128;
+  public String mFileSize = "100k";
```
Comment (yuzhu) [inline stress/common/src/main/java/alluxio/stress/jobservice/JobServiceBenchParameters.java:43]: this seems like a rather large choice?

**Review by yuzhu — COMMENTED** [review]
left a comment

File: stress/common/src/main/java/alluxio/stress/jobservice/JobServiceBenchParameters.java (hunk 1 above)
Comment (jja725) [reply stress/common/src/main/java/alluxio/stress/jobservice/JobServiceBenchParameters.java]: update to 1k for default value

**Review by yuzhu — APPROVED** [review]
LGTM

**Comment by yuzhu** [comment]
alluxio-bot, merge this please

**Pull request merged**